src/server.js

```diff
@@ -77,6 +81,68 @@ export class Server {
   /**
    * Submits a transaction to the network.
    *
+   * If you submit a `manageOffer` operation, this function will add
+   * an attribute to the response that will help you analyze what happened
+   * with your offer.
+   *
+   * Ex:
+   * ```javascript
+   * const res = {
+   *   ...response,
+   *   offerResults: [
+   *     {
+   *       // This is a list of potential offers generated from this
+   *       // manageOffer operation. All but the last one have been
+   *       // deleted. The last one _may_ be on the book, or may
+   *       // be partially filled, or may be deleted too.
+   *       offersClaimed: [
+   *         sellerId: String,
+   *         offerId: number,
+   *         assetSold: { name, value },
+   *         assetBought: { name, value }
+   *       ],
+   *
+   *       // What effect your manageOffer op had.
+   *       effect: "manageOfferCreated|manageOfferUpdated|manageOfferDeleted",
+   *
+   *       // Whether your offer immediately got matched and filled
+   *       wasImmediatelyFilled: boolean,
+   *
+   *       // Whether your offer immediately got deleted, if for example the order was too small
+   *       wasImmediatelyDeleted: boolean,
+   *
+   *       // Whether the offer was partially, but not completely, filled
+   *       wasPartiallyFilled: boolean,
+   *
+   *       // The full requested amount of the offer is open for matching
+   *       isFullyOpen: boolean,
+   *
+   *       // The total amount of tokens bought / sold during transaction execution
+   *       amountBought: number,
+   *       amountSold: number,
+   *
+   *       // if the offer was created, updated, or partially filled, this is the current ID of the outstanding offer
+   *       lastOfferId: number,
+   *
+   *       // the index of this particular operation in the op stack
+   *       index: number
```

Comment:
Instead of index can we change this to:
```js
offersClaimed: [
   null, // if op 0 wasn't manage_offer
   {
      sellerId: String,
      offerId: number,
      // ...
   }
]
```
Much easier to access the data. EDIT: it seems that it actually returns data in such format (`map`) so `index` is redundant, correct?

Reply:
I personally don't like having null entries in an array of objects, it's annoying for devs to have to check if the entry is null. Ultimately though I think either option has pros and cons, and since none seems like a clear winner to me, I'll keep things as they are.